.github/workflows/ci.yml

```diff
@@ -9,19 +9,24 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [10.x, 12.x]
+        node: [12.x]
         os: [ubuntu-latest, macOS-latest, windows-latest]
     steps:
-      - uses: actions/checkout@v1
+      - name: Checkout
+        uses: actions/checkout@v1
+
       - name: Use Node.js ${{ matrix.node }}
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node }}
           registry-url: https://registry.npmjs.org
```

Comment:
This may not be necessary

Reply:
Which part and why?

Reply:
`registry-url: https://registry.npmjs.org`  
I think this is used by default